### DISCUSSION TITLE
token method returns "CSS Variable" for semanticTokens while having multiple color mode

### DISCUSSION THREAD

**Discussion by theMasix** [chakra-ui/panda#2434] [Q&A]
### Description

**Related to https://github.com/chakra-ui/panda/issues/2346**
---

### Problem Statement/Justification

The `token` still returns the `CSS Variable` if a `semanticToken` has multiple color-mode value (`base`, `_dark`, etc.)

Reproduction Link:
https://play.panda-css.com/q6tGWtQPKV

### First Solution | Preferred | Breaking Change

It would be great if `token` method accepts a `colorMode` as the second/third argument, and if the `colorMode` is passed, the `token` method should return the actual value of a semantic token of that mode.

### Second Solution

**I mean the user must have access to the pure colors at runtime.** So the second solution would be exporting another method that contains all the pure values for every color mode.

**Comment by astahmer** [comment]
if you need the colors at runtime, it sounds like you can use `getComputedStyle` (like [here](https://github.com/chakra-ui/panda/discussions/2200#discussioncomment-8456355)) to get the exact value being used no matter the condition

can you provide give an example of what you're trying to do ?  related https://github.com/chakra-ui/panda/discussions/2421#discussioncomment-8886378
